src/vmm/src/config.rs

```diff
@@ -103,12 +109,9 @@ impl TryFrom<String> for KernelConfig {
 
     fn try_from(kernel_cfg_str: String) -> result::Result<Self, Self::Error> {
         // Supported options:
-        // `cmdline=<"string">,path=/path/to/kernel,himem_start=<u64>`
-        // Required: all
-        let options: Vec<&str> = kernel_cfg_str.split(',').collect();
-        if options.len() != 3 {
-            return Err(ConversionError::ParseKernel(kernel_cfg_str));
-        }
+        // `cmdline=<"string"> path=/path/to/kernel himem_start=<u64>`
+        // Required: path
+        let options: Vec<&str> = kernel_cfg_str.split(' ').collect();
```

Comment:
This will split the `cmdline` value in its constituents too, which will make the following split/parsing pretty difficult. Can we stick to `,`?

Reply:
Yap, that makes total sense. I completely missed that. I'll revert it to use `,`.